_deployment/checklist/012_common_configuration.md

```diff
@@ -0,0 +1,262 @@
+---
+title: Common Configuration Steps
+permalink: "deployment/checklist/commonconfiguration.html"
+layout: "document"
+categories: ["deployment", "checklist", "tds"]
+---
+
+# Common TDS Configuration Steps
+
+The following are instructions for steps for common configuration changes in TDS.
+
+## Changing Assessment Labels in TDS
+Assessment labels are defined in the `configs.client_testproperties` table. The following steps can be taken
+to update the label for a particular assessment id:
+
+1. Update the `label` column in the `config.client_testproperties` 
+```
+UPDATE configs.client_testproperties 
+SET label = '<desired assessment label>' 
+WHERE testid = '<assessment id>'; 
+``` 
+2. Flush the redis cache
+
+
+## Enabling Segment Entry/Exit Approval
+Segment entry/exit approval is disabled by default for all segments. When enabled, a student will need to be approved by a 
+proctor before entering or exiting the segment. Segment entry/exit approval can be enabled for a particular segment by 
+taking the following actions:
+
+1. Update the `entryapproval` or `exitapproval` flag for the particular segment id
+```
+UPDATE configs.client_segmentproperties 
+SET entryapproval = 1 
+WHERE segmentid = '<segment id>';
+```
+2. Flush the redis cache
+
+
+## Configuring Universal Tools/Designated Supports/Accommodations for Specific Assessments and Segments
+Accommodations, universal tools, and designated supports can be configured for specific segments or assessments. Test tool configurations
+are specific to assessment/segment ids, meaning that new versions of the same assessment id should inherit the configuration of the 
+previous year. The three tables of concern are:
+
+* `configs.client_testtooltype` - Defines a _category_ or "type" of tool (i.e., "American Sign Language")'
+* `configs.client_testtool` - Defines the tool options based on [ISAAP codes](http://www.smarterapp.org/documents/ISAAP-AccessibilityFeatureCodes.pdf) (i.e., "TDS_ASL1" means "American Sign Language" is enabled)
+    * For each tool "type", there are typically two or more rows in this table
+    * If the `dependsontooltype` column is not null, that dependency will need to be defined in `configs.client_tooldependencies`
+* `configs.client_tooldependencies` - Defines the set of dependency rules for a particular _type_ of tool, if such rules exist
+    * The tooldependency rules are represented by the following conditional expression:
+        * IF <`iftype`> IS <`ifvalue`> THEN <`thentype`> IS <`thenvalue`>
+    * For example, IF <`Language`> IS <`ENU-Braille`> THEN <`Masking`> IS <`TDS_Masking0` (disabled)> 
+        * In other words, if "Braille" is selected as the language, then the masking tool should be disabled by default
+
+The simplest way to configure accommodations/tools for a new assessment that does not have any tools configured is to select an existing 
```

Comment:
What if this is a new database that doesn't have any assessments?  This is a common case so far in our deployments.

Reply:
Regardless of whether or not there is any assessments actually uploaded into TDS, assuming that the system was setup using the `db-schema-setup.sh` script, there should be existing seed data for other assessments. I can add more info on adding new accommodations, but there is a number of flags/db columns that are a bit ambiguous and specific to certain tooltypes.

Reply:
Since I'm literally trying to debug a system right now that doesn't have accommodations loaded I think at a minimum we need to start pointing people to the scripts that load this data.  Searching the github project for the right SQL script seems like it could lead to errors.